templates/includes/hero-img.html

```diff
@@ -1,34 +1,39 @@
 {% load wagtailimages_tags %}
 {% load wagtailcore_tags %}
-{% if page.hero_image %}
+{% if hero_image %}
```

Comment:
I can think we can safely assume that hero images are only used in the context of specific pages, right? In which case, `page.hero_image` is fine? Am I missing something?

Reply:
You're right, but the logic behind this it de-couples the BE variable naming convention to the FE logic. 

For example we have two pages, one page that has a specific hero image and one that does not so we use the listing image instead. We can use the same template by just passing in the correct image the variable `hero_image`. 

Expanding the scope a little bit, if you think about the streamfield elements that are also used in pages, it allows us to use one template for both as we can map to the variable names instead of the BE properties.